migrations/versions/20210505_cad3c933830b_creating_table_for_pending_acoustic_.py

```diff
@@ -0,0 +1,55 @@
+"""Creating table for pending acoustic records
+
+Revision ID: cad3c933830b
+Revises: bd7adf1a3801
+Create Date: 2021-05-05 06:53:21.736467
+
+"""
+# pylint: disable=no-member invalid-name
+# no-member is triggered by alembic.op, which has dynamically added functions
+# invalid-name is triggered by migration file names with a date prefix
+# invalid-name is triggered by top-level alembic constants like revision instead of REVISION
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.schema import CreateSequence, Sequence
+
+# revision identifiers, used by Alembic.
+revision = "cad3c933830b"  # pragma: allowlist secret
+down_revision = "bd7adf1a3801"  # pragma: allowlist secret
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Following command to create sequence added manually.
+    op.execute(CreateSequence(Sequence("pending_id_sequence")))
```

Comment:
Looking at the database post-migration, I think this manually created ``Sequence`` is unnecessary. Postgres includes a sequence "pending_acoustic_id_seq", and uses it for the default value of ``id`` (``nextval('pending_acoustic_id_seq'::regclass)``).

<summary>
describe commands in pgsql:
</summary>
<details>
<pre>
postgres=# \d
                   List of relations
 Schema |          Name           |   Type   |  Owner   
--------+-------------------------+----------+----------
 public | alembic_version         | table    | postgres
 public | amo                     | table    | postgres
 public | amo_id_seq              | sequence | postgres
 public | api_client              | table    | postgres
 public | emails                  | table    | postgres
 public | fxa                     | table    | postgres
 public | fxa_id_seq              | sequence | postgres
 public | mofo                    | table    | postgres
 public | mofo_id_seq             | sequence | postgres
 public | newsletters             | table    | postgres
 public | newsletters_id_seq      | sequence | postgres
 public | pending_acoustic        | table    | postgres
 public | pending_acoustic_id_seq | sequence | postgres
 public | pending_id_sequence     | sequence | postgres
 public | vpn_waitlist            | table    | postgres
 public | vpn_waitlist_id_seq     | sequence | postgres
(16 rows)

postgres=# \d pending_acoustic
                                          Table "public.pending_acoustic"
      Column      |           Type           | Collation | Nullable |                   Default                    
------------------+--------------------------+-----------+----------+----------------------------------------------
 id               | integer                  |           | not null | nextval('pending_acoustic_id_seq'::regclass)
 email_id         | uuid                     |           | not null | 
 retry            | integer                  |           | not null | 
 create_timestamp | timestamp with time zone |           | not null | now()
 update_timestamp | timestamp with time zone |           | not null | 
Indexes:
    "pending_acoustic_pkey" PRIMARY KEY, btree (id)
Foreign-key constraints:
    "pending_acoustic_email_id_fkey" FOREIGN KEY (email_id) REFERENCES emails(email_id)
</pre>
</details>